hls-test-utils/hls-test-utils.cabal

```diff
@@ -44,7 +44,7 @@ library
     , ghcide                  ^>=1.4
     , hls-graph
     , hls-plugin-api          ^>=1.2
-    , hspec                   <2.8
+    , hspec                   <2.9
```

Comment:
this breaks hls-test-utils(https://github.com/haskell/haskell-language-server/runs/3494467098?check_suite_focus=true):

```
[1 of 2] Compiling Test.Hls.Util    ( src/Test/Hls/Util.hs, /home/runner/work/haskell-language-server/haskell-language-server/dist-newstyle/build/x86_64-linux/ghc-8.10.2/hls-test-utils-1.1.0.0/build/Test/Hls/Util.o, /home/runner/work/haskell-language-server/haskell-language-server/dist-newstyle/build/x86_64-linux/ghc-8.10.2/hls-test-utils-1.1.0.0/build/Test/Hls/Util.dyn_o )
Configuring executable 'hls-wrapper' for hls-1.3.0.0..

src/Test/Hls/Util.hs:176:32: error:
Error:     ‘configOutputFile’ is not a (visible) constructor field name
    |
176 |                              , configOutputFile = Right $ subdir </> "results.xml"
    |                                ^^^^^^^^^^^^^^^^
```

Reply:
After reviweing last version of hspec i've not found how to set the ouput file in the new configuration :thinking: 

https://hackage.haskell.org/package/hspec-core-2.8.3/docs/Test-Hspec-Core-Runner.html#g:2

Reply:
I would rather limit the upper bounds and say, fix later

Reply:
Has nothing to do in principle with this pr